package.json

```diff
@@ -99,6 +100,7 @@
     "rimraf": "^3.0.2",
     "ts-node": "^10.9.1",
     "typescript": "^4.7.4",
+    "vite": "^2.9.15",
```

Comment:
Is this needed since we are relying on the developer's version?

Reply:
we want it as a dev dep for the types, yes